### PR TITLE
DP-13727 Add CodeArtifact package paths to service.yml

### PR DESCRIPTION
**Pull request by hui-zheng278**
One of the Capsaicin requirements requires that all artifacts uploaded to CodeArtifacts come from specific Semaphore jobs.
This PR adds package_paths to service.yml so that each GitHub repo can declare the path of the CodeArtifact packages that it writes to.
For newer GitHub repos, the package_paths needs to be added to the service.yml file manually before it can write to CodeArtifact in the Semaphore pipeline.
For more information about this, see https://confluent.slack.com/archives/C038ZJ00P/p1708464192287999

### CODE REVIEW

File: service.yml
Comment (rahejaprince) [inline service.yml:19]: is this path same for nightlies as well as RC builds? Can you please check with owner of `cp-dockerfile` pipeline_type for this?

**Comment by rahejaprince** [comment]
This PR needs to be merged into 6.2.x and then pint merged to master

File: service.yml
Comment (hui-zheng278) [reply service.yml:19]: > is this path same for nightlies as well as RC builds?
These paths are at the repo level, as long as the builds are in this repo, they will be allowed to write to these paths.

**Comment by hui-zheng278** [comment]
> This PR needs to be merged into 6.2.x and then pint merged to master

@rahejaprince Since you are the owner of the repo, I'll leave it to you to decide how to merge.

**Comment by rahejaprince** [comment]
> > This PR needs to be merged into 6.2.x and then pint merged to master
> 
> @rahejaprince Since you are the owner of the repo, I'll leave it to you to decide how to merge.

Please change the base of this PR to 6.2.x branch and we can then pint merge it once this PR is merged. Please resolve conflicts also.

**Comment by cchristous** [comment]
> Please change the base of this PR to 6.2.x branch and we can then pint merge it once this PR is merged. Please resolve conflicts also.

This PR and the like was opened as a best effort courtesy for all repos. Please get this change into the default branch of this repo in whatever way you deem best. If this is not done so by March 25, the repo will be unable to push artifacts to CodeArtifact

**Review by hk10111 — DISMISSED** [review]

**Review by hk10111 — APPROVED** [review]

**Pull request merged**
